includes/core-changes/corefx/5.0/binaryformatter-serialization-obsolete.md

```diff
@@ -0,0 +1,92 @@
+
+The following serialization methods are also now obsolete, but have no behavioral changes:
+
+- <xref:System.Runtime.Serialization.Formatter.Serialize(System.IO.Stream,System.Object)?displayProperty=nameWithType>
+- <xref:System.Runtime.Serialization.Formatter.Deserialize(System.IO.Stream)?displayProperty=nameWithType>
+- <xref:System.Runtime.Serialization.IFormatter.Serialize(System.IO.Stream,System.Object)?displayProperty=nameWithType>
+- <xref:System.Runtime.Serialization.IFormatter.Deserialize(System.IO.Stream)?displayProperty=nameWithType>
+
+#### Version introduced
+
+5.0 Preview 8
+
+#### Recommended action
+
+- Stop using <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter> in your code. Instead, consider using <xref:System.Text.Json.JsonSerializer> or <xref:System.Xml.Serialization.XmlSerializer>. For more information, see [BinaryFormatter security guide](../../../../docs/standard/serialization/binaryformatter-security-guide.md).
+
+- You can temporarily suppress the <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter> compile-time warning, which is `SYSLIB0011`. We recommend that you thoroughly assess your code for risks before choosing this option. The easiest way to suppress the warnings is to surround the individual call site with `#pragma` directives.
+
+  ```csharp
+  // Now read the purchase order back from disk
+  using (FileStream readStream = new FileStream("myfile.bin", FileMode.Open))
+  {
+      BinaryFormatter formatter = new BinaryFormatter();
+  #pragma warning disable SYSLIB0011
+      return (PurchaseOrder)formatter.Deserialize(readStream);
+  #pragma warning restore SYSLIB0011
+  }
+  ```
+
+  You can also suppress the warning in the project file.
+
+  ```xml
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <!-- Disable "BinaryFormatter is obsolete" warnings for entire project -->
+    <NoWarn>$(NoWarn);SYSLIB0011</NoWarn>
+  </PropertyGroup>
+  ```
+
+  If you suppress the warning in the project file, the warning is suppressed for all code files in the project. Suppressing SYSLIB0011 does not suppress warnings caused by using other obsolete APIs.
+
+- To continue using <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter> in ASP.NET apps, you can re-enable it in the project file. However, it's strongly recommended not to do this. For more information, see [BinaryFormatter security guide](../../../../docs/standard/serialization/binaryformatter-security-guide.md).
+
+  ```xml
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <!-- Warning: Setting the following switch is *NOT* recommended in web apps. -->
+    <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
+  </PropertyGroup>
+  ```
+
+For more information about recommended actions, see [Resolving BinaryFormatter obsoletion and disablement errors](https://aka.ms/binaryformatter).
```

Comment:
This aka.ms link points to a file in @GrabYourPitchforks fork of docs repo. Should the file be moved in the original repo here instead?

Reply:
The aka.ms link will be updated after the final doc is published on docs.microsoft.com.